### DISCUSSION TITLE
Support Visibility in Revision

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/PublishPress-Revisions#916] [Feature Requests]
During revision, it does not allow to edit visibility status
![image](https://user-images.githubusercontent.com/83811723/194821530-0c1d2e13-8c1d-4fd5-bc04-e3040e991bf3.png)

It would be great if we support it.

From ticket: 161728

**Comment by KratosGemini** [comment]
I haven't needed this but thought I'd mention it looks similar to #905
